prices/__init__.py

```diff
@@ -324,5 +324,26 @@ def apply(self, price_obj):
                      currency=price_obj.currency, history=history)
 
 
+class PercentageDiscount(PriceModifier):
```

Comment:
What if it was called `FractionDiscount` and did not explicitly multiply the factor by `0.01`?